Framework/Algorithms/src/Stitch1DMany.cpp

```diff
@@ -168,18 +177,14 @@ std::map<std::string, std::string> Stitch1DMany::validateCommonInputs() {
     errors["InputWorkspaces"] = "At least 2 input workspaces required.";
 
   // Check that all the workspaces are of the same type
-  if (!m_inputWSMatrix.empty() && !m_inputWSMatrix[0].empty()) {
```

Comment:
The `InputWorkspaces` property was given a validator that made entering at least one workspace for it mandatory. Therefore it would not be possible for `m_inputWSMatrix` to be completely empty, so the check is unnecessary. Like `Params` the reason for attaching a validator in the first place was to make it explicit in the documentation that input workspaces were mandatory.